force-app/main/dialogue/classes/HOT_MessageHelper.cls

```diff
@@ -77,23 +79,13 @@ public with sharing class HOT_MessageHelper {
     @AuraEnabled(cacheable=false)
     public static Thread__c createThread(Id recordId, Id accountId) {
         Thread__c thread = new Thread__c();
+        thread.HOT_Request__c = recordId.getSObjectType() == 'HOT_Request__c' ? recordId : null;
         thread.CRM_Related_Object__c = recordId;
         thread.CRM_Type__c = 'HOT';
```

Comment:
Lurer på om det egentlig er Thread Type vi skal sette her, og at det er automatikk som setter Type.

You should always define a new thread type for a different process. This means one or more new values should be added to the CRM_Thread_Type__c field. Keep in mind that the amount of data in the tables are way over the LDV limit meaning you should always optimize your queries using indexed fields. For this purpose the CRM_Type__c har been created and indexed on the thread object (This is populated in a flow for ALL thread types and copies the CRM_Thread_Type__c value)